carp.studies.core/src/commonMain/kotlin/dk/cachet/carp/studies/domain/users/Account.kt

```diff
@@ -9,15 +9,15 @@ import kotlinx.serialization.Serializable
  */
 @Serializable
 data class Account(
-    /**
+        /**
      * Identities associated with this account.
      */
-    val identities: List<AccountIdentity> = listOf(),
-    /**
+    val identity: AccountIdentity,
```

Comment:
Interesting to note this means an Account can now also no longer exist without an identity. This was not the case before (unintentionally, I believe).